### PR TITLE
Feature rename Credential to EmailCredential

### PR DESCRIPTION
**Pull request by marcelobarreto**

### CODE REVIEW

File: spec/bdd/hello/management/cancel_account_spec.rb [hunk 1]
```diff
@@ -49,9 +49,10 @@
               end
 
               Then "and my data should be removed from the database" do
-                expect(User.count).to        eq(0)
-                expect(Credential.count).to  eq(0)
-                expect(Access.count).to      eq(0)
+                expect(User.count).to            eq(0)
+                expect(Credential.count).to      eq(0)
+                expect(EmailCredential.count).to eq(0)
```
Comment (thejamespinto) [inline spec/bdd/hello/management/cancel_account_spec.rb]: can you please remove this line?

**Comment by thejamespinto** [comment]
@marcelobarreto LGTM, there's only one comment

**Comment by thejamespinto** [comment]
@marcelobarreto squash those please, having reviews on the git history is lame 😄

**Comment by marcelobarreto** [comment]
Done :+1:

**Comment by thejamespinto** [comment]
closes #6

**Comment by thejamespinto** [comment]
❤️  thank you 👍

**Pull request merged**